index.html

```diff
@@ -3885,6 +3885,32 @@ <h3>JSON-LD</h3>
 base media type, `application/vc+ld+json`.
         </p>
 
+        <p>
+As elaborated upon in Section <a href="#static-processing"></a>, some software
+applications might not perform dynamic JSON-LD processing, which uses features
+such as JSON-LD compaction and expansion. Static processors can elect to reject
+unrecognized URL values in the `@context` property, for example. Authors of
+<a>conforming documents</a> are also advised that use of certain dynamic
+JSON-LD processor features might reduce interoperability. These features
+include the following:
+        </p>
+
+        <ul>
+          <li>
+Use of JSON-LD keywords in the `@context` value that globally modify document
+value processing, such as global settings of `@base`
+          </li>
+          <li>
+Use of full URLs for JSON-LD terms and types (e.g.,
+`https://www.w3.org/2018/credentials#VerifiableCredential`), instead of the
+unmapped forms of any such values (e.g., `VerifiableCredential`) that are
```

Comment:
(e.g., `https://www.w3.org/2018/credentials#VerifiableCredential` or `https://example.org/somenewtype`) instead of the unmapped forms of any such values (e.g., `VerifiableCredential` or `somenewtype`) that are

Reply:
@David-Chadwick -- "that are" what? [Your comment above](https://github.com/w3c/vc-data-model/pull/1276#discussion_r1382566565) seems to have been cut off.

Reply:
no its not. This is a replacement for lines 3905 and 3906 so that 3907 continues as before

Reply:
Thankyou. I will try this next time. I was not aware of the feature. I only thought you could edit line by line